README.md

```diff
@@ -1044,20 +1044,34 @@ wifi | booolean | no | Either to enable or disable Wi-Fi. | false
 data | booolean | no | Either to enable or disable mobile data. | false
 airplaneMode | booolean | no | Either to enable or disable Airplane Mode. | false
 
+### mobile: getAppStrings
+
+Retrieves string resources for the given app language. An error is thrown if strings cannot be fetched or no strings exist
+for the given language abbreviation.
```

Comment:
ah, need to address the supported version?

Reply:
not sure what you mean?

Reply:
I mean this is available since uia2 driver version 2.15.0

Reply:
added